WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1760,7 +1772,8 @@ public boolean updatePostContentNewEditor(boolean isAutoSave, String title, Stri
             contentChanged = true;
         } else if (mEditorFragment instanceof AztecEditorFragment
```

Comment:
Shouldn't we be checking `mMediaInsertedOnCreation` separately, like this?

```
if (mMediaInsertedOnCreation) {
    mMediaInsertedOnCreation = false;
    contentChanged = true;
} else if (compareCurrentMediaMarkedUploadingToOriginal(content)) {
    contentChanged = true;
} else if (mEditorFragment instanceof AztecEditorFragment
        && ((AztecEditorFragment) mEditorFragment).isHistoryEnabled()) {
    contentChanged = ((AztecEditorFragment) mEditorFragment).hasHistory();
} else {
    contentChanged = mPost.getContent().compareTo(content) != 0;
}
```

Reply:
oh good catch! I think I was blurred by the fact that the feature can only be there if Aztec is present, but Aztec being present != Aztec's history is enabled no no! Good catch again.
Addressed in 08aa567